cloudcix_primitives/lxd.py

```diff
@@ -148,22 +140,18 @@ def build(
 
     # Define message
     messages = {
-        1000: f'Successfully created {instance_type} {name} on {endpoint_url}',
-        3011: f'Invalid instance_type "{instance_type}" sent. Supported instance types are "containers" and "virtual_machines"',
+        1000: f'Successfully created container {name} on {endpoint_url}',
         3021: f'Failed to connect to {endpoint_url} for projects.exists payload',
         3022: f'Failed to run projects.exists payload on {endpoint_url}. Payload exited with status ',
         3023: f'Failed to connect to {endpoint_url} for projects.create payload',
         3024: f'Failed to run projects.create payload on {endpoint_url}. Payload exited with status ',
-        3025: f'Failed to connect to {endpoint_url} for {instance_type}.exists payload',
-        3026: f'Failed to run {instance_type}.exists payload on {endpoint_url}. Payload exited with status ',
-        3027: f'Failed to connect to {endpoint_url} for {instance_type}.exists payload',
-        3028: f'Failed to run {instance_type}.exists payload on {endpoint_url}. Payload exited with status ',
+        3025: f'Failed to connect to {endpoint_url} for container.exists payload',
+        3026: f'Failed to run container.exists payload on {endpoint_url}. Payload exited with status ',
```

Comment:
```suggestion
        3026: f'Failed to run containers.exists payload on {endpoint_url}. Payload exited with status ',
```

The PyLXD Client for the a container is `containers` - https://pylxd.readthedocs.io/en/latest/api.html#pylxd.client.Client.containers. 

Can you correct this in the remainder of the file?